generators/server/index.js

```diff
@@ -283,6 +279,25 @@ module.exports = JhipsterServerGenerator.extend({
                 },
                 {
                     when: function (response) {
+                        return applicationType == 'monolith' && (response.authenticationType == 'session' || response.authenticationType == 'jwt');
```

Comment:
@moifort why did you add this as a new question, the way it was done earlier seemed better to me as its a type of auth and should be part of the same question